docs/docs-beta/docs/dagster-plus/deployment/management/environment-variables/agent-config.mdx

```diff
@@ -0,0 +1,335 @@
+      same structure as the ECS API
+    </a>
+    .
+  </ReferenceTableItem>
+  <ReferenceTableItem propertyName="container_context.ecs.secrets_tags">
+    A list of tag names. Each secret tagged with any of those tag names in AWS
+    Secrets Manager will be included in the launched tasks as environment
+    variables. The name of the environment variable will be the name of the
+    secret, and the value of the environment variable will be the value of the
+    secret.
+  </ReferenceTableItem>
+</ReferenceTable>
+*/}
+
+
+After you've modified `dagster_cloud.yaml`, redeploy the code location in Dagster+ to apply the changes:
+
+<ThemedImage
+  alt="Highlighted Redeploy option in the dropdown menu next to a code location in Dagster+"
+  style={{width:'100%', height: 'auto'}}
+  sources={{
+    light: '/images/dagster-cloud/developing-testing/code-locations/redeploy-code-location.png',
+    dark: '/images/dagster-cloud/developing-testing/code-locations/redeploy-code-location.png',
+  }}
+/>
+
+
+</TabItem>
+<TabItem value="Docker">
+
+### Docker agents
+
+Using the `container_context.docker.env_vars` property, you can include environment variables and secrets in the Docker container associated with a specific code location. For example:
+
+```yaml
+# dagster_cloud.yaml
+locations:
+  - location_name: cloud-examples
+    image: dagster/dagster-cloud-examples:latest
+    code_source:
+      package_name: dagster_cloud_examples
+    container_context:
+      docker:
+        env_vars:
+          - DATABASE_NAME
+          - DATABASE_USERNAME=hooli_testing
+```
+
+The `container_context.docker.env_vars` property is a list, where each item can be either `KEY` or `KEY=VALUE`. If only `KEY` is specified, the value will be pulled from the local environment.
+
+After you've modified `dagster_cloud.yaml`, redeploy the code location in Dagster+ to apply the changes:
+
+<ThemedImage
+  alt="Highlighted Redeploy option in the dropdown menu next to a code location in Dagster+"
+  style={{width:'100%', height: 'auto'}}
+  sources={{
+    light: '/images/dagster-cloud/developing-testing/code-locations/redeploy-code-location.png',
+    dark: '/images/dagster-cloud/developing-testing/code-locations/redeploy-code-location.png',
+  }}
+/>
+
+</TabItem>
+<TabItem value="Kubernetes">
+
+### Kubernetes agents
+
+Using the `container_context.k8s.env_vars` and `container_context.k8s.env_secrets` properties, you can specify environment variables and secrets for a specific code location. For example:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: cloud-examples
+    image: dagster/dagster-cloud-examples:latest
+    code_source:
+      package_name: dagster_cloud_examples
+    container_context:
+      k8s:
+        env_vars:
+          - database_name        # value pulled from agent's environment
+          - database_username=hooli_testing
+        env_secrets:
+          - database_password
+```
+
+{/*
+<ReferenceTable>
```

Comment:
I think this will just need to be a Markdown table for now.

Reply:
I ended up using a vanilla html table since there was nested html in the content. It seems to be pretty decent.
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/e6a04a00-7cc9-47df-aa61-f5222b07d7ab" />

Reply:
Whoops, just realized that in the legacy docs it's pivoted. Let me update.
<img width="1154" alt="image" src="https://github.com/user-attachments/assets/474c1c0c-9ab0-4734-950d-dc1491dacaff" />

Reply:
Ok, ended up using markdown tables like you originally suggested. Nice!
<img width="1050" alt="image" src="https://github.com/user-attachments/assets/6ff894aa-a330-4d0b-9494-5da5456ec199" />